src/test/app/WasmHostFuncImpl_test.cpp

```diff
@@ -217,7 +217,7 @@ struct WasmHostFuncImpl_test : public beast::unit_test::suite
         WasmHostFunctionsImpl hfs(ac, dummyEscrow);
 
         // Use featureSmartEscrow for testing
-        auto const amendmentId = featureSmartEscrow;
+        auto const amendmentId = featureTokenEscrow;
```

Comment:
Why is the feature name different from the amendment name?

Reply:
This is checking for an amendment that is activated. I think the switch to default `Supported::no` messed with how the function deems amendments "valid", so I just switched it here to an amendment marked as `Supported::yes`, since that'll definitely be true of any amendment that is active here.